docs/en/ingest-management/elastic-agent/install-elastic-agent.asciidoc

```diff
@@ -51,3 +51,13 @@ Refer to:
 * {eck-ref}/k8s-elastic-agent.html[Run {agent} on ECK] -- for {eck} users
 --
 
+== Minimum Requirements
+
+An example of the {agent} resource consumption is below.
```

Comment:
The statement here seems too noncommittal for docs. The content doesn't live up to the expectation set by the heading.

We should start with a clear statement of minimum system requirements. If it's not possible to make a clear statement, then I think we need to do more testing before we try to document these requirements. IMO if the requirements vary by platform, then we need to provide a breakdown by platform in table format.

Also we need to provide more guidance about how much extra memory is consumed when you add integrations. Otherwise I feel this content is not super useful and potentially misleading.

Can you recast this section so that it 1) Makes a direct statement about minimum system requirements, and 2) provides an example of system resources used by a specific instance?